modules/instrument_builder/test/instrument_builderTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Instrument_builder automated integration tests
+ *
+ * PHP Version 5
+ *
+ * @category Test
+ * @package  Loris
+ * @author   Ted Strauss <ted.strauss@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once __DIR__ . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
+class instrumentBuilderTestIntegrationTest extends LorisIntegrationTest
+{
+    protected $url = 'http://localhost/main.php?test_name=instrument_builder';
```

Comment:
This hides the variable from the LorisIntegrationTest class which looked up the URL from the config settings.

Can you remove this class variable and instead call get as:

$this->webDriver->get($this->url . "?test_name=instrument_builder");